common/persistence/visibility/store/query/converter.go

```diff
@@ -592,13 +592,18 @@ func ParseSqlValue(sqlValue string) (interface{}, error) {
 	return nil, NewConverterError("%s: unable to parse %s", InvalidExpressionErrMessage, sqlValue)
 }
 
-func convertColName(fnInterceptor FieldNameInterceptor, colNameExpr sqlparser.Expr, usage FieldNameUsage) (string, error) {
+func convertColName(fnInterceptor FieldNameInterceptor, colNameExpr sqlparser.Expr, usage FieldNameUsage) (alias string, fieldName string, err error) {
 	colName, isColName := colNameExpr.(*sqlparser.ColName)
 	if !isColName {
-		return "", NewConverterError("%s: must be a column name but was %T", InvalidExpressionErrMessage, colNameExpr)
+		return "", "", NewConverterError("%s: must be a column name but was %T", InvalidExpressionErrMessage, colNameExpr)
 	}
 
 	colNameStr := sqlparser.String(colName)
 	colNameStr = strings.ReplaceAll(colNameStr, "`", "")
-	return fnInterceptor.Name(colNameStr, usage)
+	name, err := fnInterceptor.Name(colNameStr, usage)
```

Comment:
nit: just to be clear
```suggestion
	fieldName, err := fnInterceptor.Name(colNameStr, usage)
```